department-of-veterans-affairs/va.gov-cms
# VACMS-5980: Add address line 2 to entity types.

## Description
Closes #5980

## Testing done
Visual / behat

## QA steps
 - As an administrator, go to `admin/structure/types/manage/vet_center_cap/fields/node.vet_center_cap.field_address` and `admin/structure/paragraphs_type/service_location_address/fields/paragraph.service_location_address.field_address` and visually verify address line 2 is set to optional.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
